models/articlesModels.js

```diff
@@ -45,3 +44,20 @@ exports.patchIncreaseVotes = (id, vote) => {
     [vote, id]
   );
 };
+
+exports.articleIdWithComment = (id) => {
+  return db
+    .query(
+      "SELECT comment_id, body, author, votes, created_at FROM comments WHERE article_id = $1;",
+      [id]
+    )
+    .then(({ rows }) => {
+      if (!rows.length) {
+        return Promise.reject({
```

Comment:
this is unsual, you don't need to reject this, getting back an empty array is a valid response. Think how you can check if an article does exist.
hint: Promise.all in the controller